cmake/SetupCompilerOptions.cmake

```diff
@@ -111,6 +111,20 @@ if(BLT_DEFINES)
     message(STATUS "Added \"${BLT_DEFINES}\" to definitions")
 endif()
 
+if(COMPILER_FAMILY_IS_MSVC)
+    add_definitions(-D_USE_MATH_DEFINES)
+
+    # Mark 32 bit executables large address aware so they can use > 2GB address space
```

Comment:
I haven't run into these issues with VS 2015 or 2017, but I have been doing 64-bit builds.

Appveyor is using 2015, and seems ok with these.  Could be harmless, what do folks think?

Kathleen Biagas from the VisIt team may have some wisdom -- I believe we have stopped building 32-bit versions of VisIt not to long ago.

For context, on VisIt, we use VS 2012 and VS 2013.
(here are the lib sets and projects for our tpls http://visit.ilight.com/svn/visit/trunk/windowsbuild/)